app/src/tunnel/kotlin/tunnel/Persistence.kt

```diff
@@ -21,32 +21,42 @@ class RulesPersistence {
     }
     val save = { id: FilterId, ruleset: Ruleset ->
         Result.of { core.Persistence.paper().write("rules:set:$id", ruleset) }
-        Result.of { core.Persistence.paper().write("rules:size:$id", ruleset.size) }
+        Result.of { core.Persistence.paper().write("rules:size:$id", ruleset.size)
+        }
     }
     val size = { id: FilterId ->
         Result.of { core.Persistence.paper().read("rules:size:$id", 0) }
     }
 }
+//fun WildCardIt(args: Array<String>) {
+//    val str = "Kotlination.com = Be Kotlineer - Be Simple - Be Connective"
+//
+//    val separate1 = str.split("=|-".toRegex())
+//    Log.d("Test123", ""+ seperate1)
+//}
 
 class FiltersPersistence {
     val load = { ktx: AndroidKontext ->
         loadLegacy34(ktx)
                 .or { loadLegacy35(ktx) }
                 .or {
                     ktx.v("loading from the persistence", core.Persistence.paper().path)
-                    Result.of { core.Persistence.paper().read("filters2", FilterStore()) }
```

Comment:
Yes it is because everybody who upgrades from 3.7 to a release using this code would loose all there settings. if you **realy** need to rename this you would need to add a `loadLegacy()` function for 3.7 and all other versions using `"filters2"`.